### PR TITLE
chore: cancel previous run in workflow

### PR DESCRIPTION
**Pull request by imranbarbhuiya**
**Please describe the changes this PR makes and why it should be merged:**
Cancel previously running tests when a new commit gets added.
**Status and versioning classification:**

### CODE REVIEW

**Comment by ImRodry** [comment]
You can just use this https://docs.github.com/en/actions/using-jobs/using-concurrency

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -5,6 +5,11 @@ jobs:
     name: Lint
     runs-on: ubuntu-latest
     steps:
+      - name: Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.9.1
```
Comment (ArjunSharda) [inline .github/workflows/test.yml:9]:
```suggestion
        uses: styfle/cancel-workflow-action@0.10.0
```

File: .github/workflows/documentation.yml [hunk 2]
```diff
@@ -22,6 +22,11 @@ jobs:
       BRANCH_OR_TAG: ${{ steps.env.outputs.BRANCH_OR_TAG }}
       SHA: ${{ steps.env.outputs.SHA }}
     steps:
+      - name: Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.9.1
```
Comment (ArjunSharda) [inline .github/workflows/documentation.yml:26]:
```suggestion
        uses: styfle/cancel-workflow-action@0.10
```

**Review by ArjunSharda — CHANGES_REQUESTED** [review]
I've added two suggestions, as the current version you are proposing for is outdated.

**Comment by iCrawl** [comment]
I'll close this for now, this might conflict tags and branch docs. And I think we don't have a huge issue with too many CIs running.

**Pull request closed without merging**